src/checks/lowercase_key.rs

```diff
@@ -19,14 +21,22 @@ impl Check for LowercaseKeyChecker {
         if key.to_uppercase() == key {
             None
         } else {
+            let warning_text = self.template.replace("{}", &key);
```

Comment:
You can move this line to the `message` function. For example:
https://github.com/mgrachev/dotenv-linter/blob/master/src/checks/unordered_keys.rs#L16